src/components/PageTitle.tsx

```diff
@@ -22,13 +22,13 @@ export const PageTitle: React.FC<Props> = ({
     </PageHeader>
   ) : hasData ? (
     <PageHeader>
-      <H2 data-cy="page-title">
-        <span>
+      <Subtitle>
+        <span data-cy="page-title">
```

Comment:
i am suspicious of generic data-cy values like this. i know that this is a page and therefore the chance of it being rendered with another component that has a page title is rare. but name spacing cy attributes likes this seems safer.

Reply:
That value  was always there i just moved it. but i think this component is generic enough where i think it isnt necessary to name space it, it reflects what component it belongs too. What would you suggest name spacing it too?

Reply:
sounds good to me